src/client/pages/Download/components/DownloadPageContent.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { useTextKeys } from 'utils/textKeys'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { useBreakpoint } from 'utils/hooks/useBreakpoint'
+import { TOP_BAR_HEIGHT } from 'components/TopBar'
+import { DownloadHeadline } from './DownloadHeadline'
+import { GetAppButtons } from './GetAppButtons'
+import { AppImage } from './AppImage'
+
+const IMAGE_WIDTH = 560
+
+const Page = styled.div`
+  background: ${colorsV3.gray900};
+  max-width: 100vw;
```

Comment:
Did you get any overflow issues, or why do we need to set a max-width here? Or maybe set width `100%`

Reply:
I don't remember, it might just have been that I have experience of overflow issues when just setting width to `100%` on elements with parent elements with widths that are not defined.